test/callofcactus/DefaultMapTest.java

```diff
@@ -7,14 +8,14 @@
 /**
  * Created by guushamm on 2-11-15.
  */
-public class MapTest extends TestCase {
+public class DefaultMapTest extends TestCase {
 
-	private Map map;
+	private DefaultMap defaultMap;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid unused private fields such as 'defaultMap'.](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1385244509/issues/source?bid=2979575&fileBranchId=2980149#l13)